ddtrace/internal/datadog/profiling/stack_v2/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ if(NOT Threads_FOUND OR NOT CMAKE_USE_PTHREADS_INIT)
 endif()
 
 # Add echion
-set(ECHION_COMMIT "b2f2d49f2f5d5c4dd78d1d9b83280db8ac2949c0" CACHE STRING "Commit hash of echion to use")
+set(ECHION_COMMIT "ed744987f224fae3f93c842b2b5ffb083984ff8b" CACHE STRING "Commit hash of echion to use")
```

Comment:
What is this?

Reply:
Sorry--good catch.  I grabbed this [from main](https://github.com/DataDog/dd-trace-py/blob/main/ddtrace/internal/datadog/profiling/stack_v2/CMakeLists.txt#L40).  So, gcc 12 and earlier would (erroneously) include the `algorithm` header alongside some other headers.  Subsequently, we had a "bug" where we _omitted_ that header in one of the out-of-repo code assets--but it didn't matter in gcc 12.

In gcc 14, the underlying issue was fixed, causing our code to fail to build.  This commit fixes that.  This is spelled out better in main, but for here I tried to do the minimum change.